docs/src/Touchable.doc.js

```diff
@@ -338,7 +348,70 @@ function DisabledEx() {
         </Touchable>
       </Box>
       <Box id="count-panel" role="region" padding={2}>
-        <Text>Number of clicks: {clickCount}</Text>
+        <Text>Number of touches: {clickCount}</Text>
+      </Box>
+    </Box>
+  );
+}
+`}
+  />
+);
+
+card(
+  <Combination
+    id="tapStyleCombinations"
+    name="Combinations: tapStyle"
+    description={`\`tapStyle\` "compress" scales down by 1% on click / touch.`}
+    tapStyle={['none', 'compress']}
+  >
+    {props => (
+      <Box padding={2} width={150}>
+        <Touchable {...props} rounding={2}>
```

Comment:
@jackhsu978 could we make this `rounding="circle"`?

Before
![image](https://user-images.githubusercontent.com/127199/84518371-7910e680-ac85-11ea-9f9f-b5d1a2719b5d.png)

After
![image](https://user-images.githubusercontent.com/127199/84518502-a2317700-ac85-11ea-94a7-a6cc8802587b.png)

Reply:
good catch! I purposely wanted the rounding to _not_ match (which is generally a bad thing to do) just so that I can show the light gray background tapStyle, but that is no longer relevant. Will update.